src/asset_manager/view/AssetsView.js

```diff
@@ -56,7 +56,7 @@ export default Backbone.View.extend({
     const url = input.value.trim();
     const handleAdd = this.config.handleAdd;
 
-    if (!url) {
+    if (!url || !input) {
```

Comment:
mmm this `!input` check doesn't make sense here, it that case it would crash at `input.value.trim()`